app/controllers/results_controller.rb

```diff
@@ -134,6 +133,19 @@ def edit
     end
   end
 
+  def run_tests
+    grouping_id = params[:grouping_id]
+    begin
+      AutomatedTestsHelper.request_a_test_run(grouping_id,
+                                              'request',
```

Comment:
Let's use `'submission'` instead. (Note that the tests will have to be enabled with "run on submission"). This will be something which differentiates a student and instructor running tests.